cli/list.go

```diff
@@ -51,12 +53,17 @@ func workspaceListRowFromWorkspace(now time.Time, usersByID map[uuid.UUID]coders
 		}
 	}
 
+	healthy := ""
```

Comment:
Technically, it could be covered with unit tests.

Reply:
Testing an individual column feels a bit overkill, I think this would be a good fit for a golden-file type test (we could do that for multiple commands).

Essentially, set up a server, some workspace and agents, a list of commands to run, and generate a golden file for each command.

Reply:
I'm fine with leaving it as is.